full-service/src/bin/main.rs

```diff
@@ -24,7 +24,7 @@ use std::{
     process::exit,
     sync::{Arc, RwLock},
 };
-use structopt::StructOpt;
+use clap::Parser;
```

Comment:
Nit: sort alphabetically 
As a side note, I am surprised CI is not complaining about this...
Running `cargo fmt` should fix it for you (but since apparently it is not enforced at CI, it might result in a lot of changes)

Reply:
fixed